lms/djangoapps/letstudy_api/views.py

```diff
@@ -37,18 +42,22 @@ def post(self, request):
         data = request.data
         email = data.get('email')
         username = data.get('username')
-        uid = data.get('uid')
+        uid = data.get('uid', None)
```

Comment:
None is the default value of the get method, I would omit to add useless default param.